frontend/views/components/LinkToCopy.vue

```diff
@@ -47,14 +47,25 @@ export default ({
     }
   },
   methods: {
+    // FIXME: probably needs to be defined centrally somewhere else
+    isMacSafari (userAgent) {
```

Comment:
Does it make sense to pass in the user agent as a parameter? Shouldn't it just directly check against `navigator.userAgent` in here?

Reply:
mainly due to the compound conditions above, having a function at the very least allows us to relocate it later, i definitely dont think embedding it in the vuejs is the best in terms of usability at least

Reply:
Sorry, I'm not following, what compound conditions above?

Reply:
safari == true &&  (edge || chrome) == false

Reply:
youre right, i think initially i wanted to have a more general two parameter function not an `isSpecificA` `isSpecificB` which is why i left the `userAgent` param there

Reply:
changed